apps/minifront/src/state/dutch-auction/index.ts

```diff
@@ -117,13 +117,13 @@ export const createDutchAuctionSlice = (): SliceCreator<DutchAuctionSlice> => (s
   txInProgress: false,
 
   auctionInfos: [],
-  loadAuctionInfos: async () => {
+  loadAuctionInfos: async (queryLatestState = false) => {
```

Comment:
In what case would we want to not query latest state?

Reply:
We generally don't want to query latest state unless the user explicitly chooses to. Otherwise, we'd be making a request to the fullnode every time the user visits the auction page, which leaks some data about their possible intentions.